src/registries/nfts_registry.ts

```diff
@@ -116,16 +114,47 @@ export const getUserCollectionTokens = async (
   }
 };
 
+const standardNormaliser = ({
+  standard
+}: {
+  standard: string
+}) => {
+  const hasDip721Term = (() => {
+    const userStandardNormalised = standard.toUpperCase();
+    const systemStandardNormalised = NFTStandard.dip721.toUpperCase();
+    const startsWithDip721 = userStandardNormalised.startsWith(systemStandardNormalised);
+    const hasSuffix = userStandardNormalised.split(systemStandardNormalised).length > 1;
+
+    return startsWithDip721 && hasSuffix;
+  })();
```

Comment:
I'd move this out of the `standardNormaliser` function as a utils like
```
const hasDip721Term = (standard) => {
    const userStandardNormalised = standard.toUpperCase();
    const systemStandardNormalised = NFTStandard.dip721.toUpperCase();
    const startsWithDip721 = userStandardNormalised.startsWith(systemStandardNormalised);
    const hasSuffix = userStandardNormalised.split(systemStandardNormalised).length > 1;
    return startsWithDip721 && hasSuffix;
};
```

Reply:
Thought about, but since there's nowhere else it's used, didn't bother.